apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,43 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
+  // Hand one is a variable that will just have the word Rock Paper or Scissor in it!
+
+  //Spec 1 - If it's a tie, return "It's a tie!" It must be exactly "It's a tie!" for the tests to pass
+
 
 function rockPaperScissors(hand1, hand2) {
+  if (hand1.toLowerCase() === hand2.toLowerCase()) {
+    return "It's a tie!";
+  }
+
+  if (hand1.toLowerCase() === 'rock') {
```

Comment:
All of your if/else if/else statements are written and indented well make it very easy to read and understand. Nice work!